%YAML:1.0

# Topic name in rosbag
PointCloudTopic: "/livox/lidar"
ImageTopic: "/camera/color/image_raw"

ExtrinsicMat: !!opencv-matrix
  rows: 4
  cols: 4
  dt: d
  data: [0.0,   -1.0,   0.0,    1.0,
            0.0,  0.0,  -1.0,    0.0,
            1.0,   0.0,    0.0,    0.0,
         0.0,   0.0,    0.0,    1.0]
# Params for Canny Edge Extraction

# Canny.gray_threshold: 5
# Canny.len_threshold: 500

# # Params for Voxel Cutting & Plane Fitting & Edge Extraction
# # Voxel.size: 1.5
# Voxel.size: 0.5 # 0.5  for ftm pillar
# # Voxel.down_sample_size: 0.3
# Voxel.down_sample_size: 0.03
# Plane.min_points_size_in_one_plane: 30
# Plane.normal_theta_min: 45
# Plane.normal_theta_max: 135
# Plane.max_size: 8
# Ransac.dis_threshold:  0.06 #0.02
# Edge.min_dis_threshold: 0.03  #0.03
# Edge.max_dis_threshold: 0.06 # 0.06



Canny.gray_threshold: 5
Canny.len_threshold: 200

# Params for Voxel Cutting & Plane Fitting & Edge Extraction
Voxel.size: 1.5
Voxel.down_sample_size: 0.00002
Plane.min_points_size: 3
Plane.normal_theta_min: 10
Plane.normal_theta_max: 160
Plane.max_size: 20
Ransac.dis_threshold: 0.2
Edge.min_dis_threshold: 0.03
Edge.max_dis_threshold: 0.5
